Python/Product/Analysis/Infrastructure/TaskExtensions.cs

```diff
@@ -17,7 +17,7 @@
 using System.Threading.Tasks;
 
 namespace Microsoft.PythonTools.Analysis.Infrastructure {
-    static class TaskExtensions {
+    public static class TaskExtensions {
```

Comment:
Does this need to be public? InternalsVisibleTo is preferable.

Reply:
They are generally useful extensions but up to you. @AlexanderSher - your opinion?

Reply:
Made internal